compiler/rustc_middle/src/query/mod.rs

```diff
@@ -293,6 +293,13 @@ rustc_queries! {
         separate_provide_extern
     }
 
+    /// Returns the const of the RHS of a const item.
```

Comment:
give some info about what it's used for and that it can only be used on type consts and will ICE otherwise

Reply:
As I was addressing your above comment I actually did this before I noticed it haha.